pkg/exchange/okex/exchange.go

```diff
@@ -411,6 +416,128 @@ func (e *Exchange) QueryOrderTrades(ctx context.Context, q types.OrderQuery) ([]
 	if errs != nil {
 		return nil, errs
 	}
+	return trades, nil
+}
+
+/*
+QueryClosedOrders can query closed orders in last 3 months, there are no time interval limitations, as long as until >= since.
+Please Use lastOrderID as cursor, only return orders later than that order, that order is not included.
+If you want to query orders by time range, please just pass since and until.
+If you want to query by cursor, please pass lastOrderID.
+Because it get correct response even pass all parameters with right time interval and invalid lastOrderID, like 0.
+Time interval boundary unit is second.
+since is inclusive, ex. order created in 1694155903, get response if query since 1694155903, get empty if query since 1694155904
+until is not inclusive, ex. order created in 1694155903, get response if query until 1694155904, get empty if query until 1694155903
+*/
+func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, since, until time.Time, lastOrderID uint64) ([]types.Order, error) {
+	if symbol == "" {
+		return nil, ErrSymbolRequired
+	}
+
+	if err := tradeRateLimiter.Wait(ctx); err != nil {
```

Comment:
Perhaps we need to adjust  token limit to avoid rate limiter error

https://www.okx.com/docs-v5/en/#order-book-trading-trade-get-order-history-last-3-months
```Rate Limit: 20 requests per 2 seconds```

you declare tradeRateLimiter  = rate.NewLimiter(rate.Every(100*time.Millisecond), 5) and used in QueryClosedOrders and QueryTrades

Reply:
Update, 相當於打了三次，我改成 300 milli seconds, 應該可以